main.tf

```diff
@@ -216,7 +238,7 @@ resource "aws_db_subnet_group" "default" {
 
 resource "aws_rds_cluster_parameter_group" "default" {
   count       = module.this.enabled ? 1 : 0
-  name        = module.this.id
+  name_prefix = "${module.this.id}${module.this.delimiter}"
```

Comment:
@Nuru As I couldn't find you on Slack I'll have to ask here:

I noticed this particular change and was wondering why it was switched from `name` to `name_prefix`? 
This change resulted in new parameter groups & option groups which were named `project-dev-db` before the change and are now named `project-dev-db-20210223080217199000000002`.